benchmarks/embeddings/bench_embeddings.py

```diff
@@ -0,0 +1,113 @@
+import time
+from typing import List, Optional
+
+from llama_index import SimpleDirectoryReader
+from llama_index.embeddings import OpenAIEmbedding
+from llama_index.embeddings.base import DEFAULT_EMBED_BATCH_SIZE
+from langchain.embeddings.huggingface import HuggingFaceEmbeddings
+from llama_index import LangchainEmbedding
+
+
+def generate_strings(num_strings: int = 100, string_length: int = 10) -> List[str]:
+    content = (
+        SimpleDirectoryReader("../../examples/paul_graham_essay/data")
+        .load_data()[0]
+        .get_content()
+    )
+    content_length = len(content)
+
+    strings_per_loop = content_length / string_length
+    strings = []
+
+    for offset in range(0, int(num_strings / strings_per_loop) + 1):
+        ptr = offset
```

Comment:
Curious, what does `ptr` stand for? pointer? Just trying to understand how this loop is working lol

Reply:
oh, it's just looping over the string. I guess we could implement it using range step size instead